changelog/2020/APRIL.md

```diff
@@ -0,0 +1,15 @@
+| Module                  | Version       |
```

Comment:
Hi @urskog84 ,

No need to update this file, it is only updated when we officially release it. 
https://github.com/CiscoTestAutomation/genieparser/blob/master/changelog/undistributed.rst is the only changelog you need to update when you're creating any new parsers or fix any issues. 

Thanks